activelearning.py

```diff
@@ -18,41 +18,76 @@ class ActiveLearning:
     """
     AL Global pipeline : so far it is a class (the config is easily passed on to all
     methods), very simple, but it can be a single simple function
+
+    Bao's comments:
+    assume we have a config object that we can pass to all the components of the
+    pipeline like in the previous code, eg "config_test.yaml"
+    setup function that creates the directories to save data, ...'
+
     """
 
     def __init__(self, config):
-        # assume we have a config object that we can pass to all the components of the
-        # pipeline like in the previous code, eg "config_test.yaml"
-        self.config = OmegaConf.load(config)
-        # setup function that creates the directories to save data, ...
+        self.config = config
         self.setup()
-        # util class to handle the statistics during training
         self.logger = Logger(self.config)
-        # load the main components of the AL pipeline
         self.oracle = Oracle(self.config, self.logger)
         self.proxy = Proxy(self.config, self.logger)
         self.acq = AcquisitionFunction(self.config, self.proxy)
-        self.env = Env(self.config, self.acq)
-        self.gflownet = GFlowNet(self.config, self.logger, self.env)
-        self.querier = Querier(self.config, self.gflownet)
+        self.gfn_agent = GflownetAgent(self.config, self.logger, self.acq)
 
     def run_pipeline(self):
-        self.iter = None
```

Comment:
Why remove the initialization self.iter ?

Reply:
I thought we don't need it, do we?